packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -340,6 +341,9 @@ export class Resolver {
     isAddPendingDocument: boolean
     isCollectionSpecific: boolean
   }) => {
+    const doc = await this.getDocument(realPath)
+
+    const oldDoc = doc?._rawData || {}
```

Comment:
Why aren't we able to use `_values` for this?

Reply:
I didn't think of using `_values` but I just tested it out and it looks like `_values` does not have the additional fields. 

For example:
`_values`
```js
{
     _collection: 'posts',
    _template: 'posts',
    title: 'Just Another Blog Post asdf',
    heroImg: undefined,
    excerpt: { type: 'root', children: [Array] },
    author: 'content/authors/napolean.md',
    date: '2021-07-12T07:00:00.000Z',
    _body: { type: 'root', children: [Array] }
}
```

`_rawvalues`

```diff
{
+  foo: 'bar',
    title: 'Just Another Blog Post asdf',
    excerpt: 'Lorem markdownum evinctus ut cape adhaeret gravis licet progenies ut haesit maxima ille. Est scorpius, mori vel in visaeque Haemoniis viperei furoris e ad vasti, distulit. Crudus sub coniuge iam: dea propera sive\n',
    author: 'content/authors/napolean.md',
    date: '2021-07-12T07:00:00.000Z',
    _body: '\n' +
      '# Vote For Pedro\n' +
      '\n' +
      'Hello, the current date and time is <DateTime format="iso" />. Quis semper [vulputate](https://example.com) aliquam venenatdis egestas sagittis quisque orci. Donec commodo sdit viverra aliquam porttitor ultrices gravida eu. Tincidunt leo, elementum mattis elementum ut nisl, justo, amet, mattis. Nunc purus, diam cdommodo tincidunt turpis. Amet, duis sed elit interdum dignissim\n' +
      '\n' +
      '![](http://res.cloudinary.com/deuzrsg3m/image/upload/v1634858384/tina-upload_yxevty.jpg)\n' + "..."
     _collection: 'posts',
    _keepTemplateKey: false,
    _template: 'posts',
    _relativePath: 'anotherPost.mdx',
    _id: 'content/posts/anotherPost.mdx'
}
```